src/bitmap/backend/atomic_bitmap.rs

```diff
@@ -37,6 +37,15 @@ impl AtomicBitmap {
         }
     }
 
+    /// Enlarge this bitmap with enough bits to track `additional_size` additional bytes at page granularity.
+    /// New bits are initialized to zero.
+    pub fn enlarge(&mut self, additional_size: usize) {
```

Comment:
It's an interesting choice to make it the size to increase rather than the new size - does that mean the caller of the API has to track the size (or can it be obtained from this API?)

Reply:
Maybe add a `::size()` to allow access to the current size?

Reply:
This API assumes the caller wants to enlarge the bitmap where new size may mean t reduce the size of the bitmap. The struct does not give the size in bytes but rather it give the len where len is the total size in bits. Could you please clarify if anything needed more here?

Reply:
You could generate an error if the size was smaller. My point is that you don't want the caller to have to track what they created the bitmap with originally or what the last enlargement they made was. Either you need to provide an API call so that the caller can work out the difference themselves or have it take the new value.